blackbox/docs/general/dql/selects.rst

```diff
@@ -613,7 +614,7 @@ for operators
 
  - all other comparison operators (excluding ``IS NULL`` and ``IS NOT NULL``)
 
-.. TIP::
+.. NOTE::
```

Comment:
This was changed from `NOTE` to `TIP` in my previous commit instead of https://github.com/crate/crate/pull/7440/files#diff-49fbb0a28e660b60f4128d1591c5fcc0L603, reverting it.